otx/cli/tools/train.py

```diff
@@ -69,8 +69,12 @@ def parse_args():
     parser = argparse.ArgumentParser()
     if not os.path.exists("./template.yaml"):
         parser.add_argument("template")
-    parser.add_argument("--data", required=False, default="./data.yaml")
```

Comment:
@cih9088  Due to the modification of the code, the otx-workspace (data.yaml created through otx build) cannot be used. (I'm sorry for the confusion because this don't have a proper test case.)
Can I revert this? Will there be any problems elsewhere?

First, I made a PR. Please confirm.
https://github.com/openvinotoolkit/training_extensions/pull/1530